test/WebJobs.Extensions.RabbitMQ.Tests/RabbitMQConfigurationTests.cs

```diff
@@ -97,11 +97,12 @@ public void Handles_Null_Attributes_And_Options(string attrHostname, string attr
         //    };
 
         //    ILoggerFactory loggerFactory = new LoggerFactory();
-        //    ILogger<RabbitMQAsyncCollector> logger = loggerFactory.CreateLogger<RabbitMQAsyncCollector>());
+        //    ILogger<RabbitMQAsyncCollector> logger = loggerFactory.CreateLogger<RabbitMQAsyncCollector>();
         //    var collector = new RabbitMQAsyncCollector(context, logger);
 
-        //    byte[] msg = Encoding.UTF8.GetBytes("hi");
-        //    await collector.AddAsync(msg);
+        //    await collector.AddAsync(Encoding.UTF8.GetBytes("hi"));
+
+        //    Assert.Equal(collector.Queue.MessageCount, (uint)5);

```

Comment:
Still having issues figuring out how to write a unit test for the message count in the queue - seems that the message count of a queue with the same queue name and host name persists, regardless of where connection logic is